app/components/bookmarks/button_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="flex items-center justify-center" id="bookmark-button">
+  <%= button_to create_or_destroy_path,
+                form_class: 'w-full h-full',
+                method: bookmarked? ? :delete : :post,
```

Comment:
Would have preferred to keep the logic in the VC class with the rest of it, but putting it in a method didn't work for some reason 🤷